### DISCUSSION TITLE
Input Selection for Remote Client

### DISCUSSION THREAD

**Discussion by YourSandwich** [LizardByte/community#330] [Sunshine]
Hello, 
I wanted to share a Game (Baldurs Gate 3) with a friend, he has low specs, so he would connect to me.

What I have done is running the game in two Separate Wine Prefixes, and Created a separate Mouse Pinter and Keyboard for him using the following commands on Linux:

```
xinput create-master second
xinput reattach 26 19 # Attatches secondary mouse to secondary Virtual mouse
xinput reattach 25 20 # Attatches secondary keyboard to secondary Virtual keyboard
```
With these commands I got two separate Groups of input devices working separately, this means two games could be played at the same time without input interfering.

But Sunshine has not an option to select which input to use for the client, so the clients connects to the primary pointer.

Would it be possible to implement this into sunshine? 
It only has to grep the (master pointer/master keyboard) and provide a way to select to which sunshine should get attached.

**Comment by YourSandwich** [comment]
I just checked, and I can manually reattach the Virtual Input devices to secondary master and it works.
It would be nice if Sunshine would also have the option to create more than one set of virtual devices.

With this case, I could for example create virtual screens or even Linux Multi Seats, and have multiple games running for multiple users. But this would require also a mechanic to select which monitor to connect to.

**Comment by darkguy2008** [comment]
Almost a year... Have my upvote! Hopefully it will gain some attention soon...
